<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wifi</class>
 <widget class="QMainWindow" name="Wifi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wifi</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" rowspan="2">
     <widget class="QPlainTextEdit" name="log"/>
    </item>
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="output">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="centerOnScroll">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="input"/>
    </item>
    <item row="0" column="2" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnTest">
        <property name="text">
         <string>Test</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnApList">
        <property name="text">
         <string>AP List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnGetIpAddress">
        <property name="text">
         <string>Get IP Address</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCheckConnection">
        <property name="text">
         <string>Check connection
with Access Point</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnConnect">
        <property name="text">
         <string>Connect to
Access Point</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDisconnect">
        <property name="text">
         <string>Disconnect from
Access Point</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStartServer">
        <property name="text">
         <string>Start TCP server</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>828</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
